packages/python-requests/build

```diff
@@ -3,4 +3,5 @@ source /opt/mesosphere/environment.export
 export LIB_INSTALL_DIR="$PKG_PATH/lib/python3.4/site-packages"
 mkdir -p "$LIB_INSTALL_DIR"
 
-pip3 install --no-deps --install-option="--prefix=$PKG_PATH" --root=/ /pkg/src/$PKG_NAME
+cd /pkg/src/$PKG_NAME
+PYTHONPATH=$PYTHONPATH:$LIB_INSTALL_DIR easy_install --prefix=$PKG_PATH .
```

Comment:
Why do we need to do this? Why isn't the pypi one good enough?

This also isn't one of the standard python install scripts we have with this change. `easy_install` is also deprecated I believe. Things are supposed to always use pip + wheel.